src/app/cube/user-profile/gravatar-image/gravatar-image.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'clark-gravatar-image',
+  template: `
+        <div id="container">
+            <img [src]="getGravatarImage()">
```

Comment:
getGravatarImage method is commented out in the component class, so it does not exist.

Reply:
So this method was moved to user profile component. Is gravatar-image.component.ts still in use? If not please remove.